config/webpack.dev.js

```diff
@@ -22,6 +22,7 @@ const PORT = process.env.PORT || 3000;
 const HMR = helpers.hasProcessFlag('hot');
 // if env is 'inmemory', the inmemory debug resource is used
 const API_URL = process.env.API_URL || (ENV==='inmemory'?'app/':'http://localhost:8080/api/');
+const STACK_API_URL = process.env.STACK_API_URL || (ENV==='inmemory'?'app/':'http://localhost:8080/api/');
```

Comment:
yes conditional checking is not needed and for using STACK_API_URL, it needs to be exported like we do for API_URL.